skiko/src/jvmMain/cpp/common/interop.cc

```diff
@@ -495,11 +523,11 @@ namespace skija {
             if (rectInts == nullptr)
                 return std::unique_ptr<SkIRect>(nullptr);
             else {
-                jint *ints = env->GetIntArrayElements(rectInts, NULL);
+                jint *ints = env->GetIntArrayElements(rectInts, 0);
                 auto result = std::unique_ptr<SkIRect>(new SkIRect{
                     ints[0], ints[1], ints[2], ints[3]
                 });
-                env->ReleaseIntArrayElements(rectInts, ints, NULL);
+                env->ReleaseIntArrayElements(rectInts, ints, 0);
```

Comment:
Here indeed `void (JNICALL *ReleaseIntArrayElements)
      (JNIEnv *env, jintArray array, jint *elems, jint mode)` where mode is documented in 
https://docs.oracle.com/javase/7/docs/technotes/guides/jni/spec/functions.html as 
```
0
copy back the content and free the elems buffer
JNI_COMMIT
copy back the content but do not free the elems buffer
JNI_ABORT
free the buffer without copying back the possible changes
```